### PR TITLE
cli examples - minor fixes

### PR DESCRIPTION
**Pull request by lavianalon**
Additional use cases for CLI v2

### CODE REVIEW

File: docs/Researcher/cli-reference/new-cli/cli-examples.md [hunk 1]
```diff
@@ -107,22 +107,25 @@ runai  mpi submit dist1 --workers=2 -g 1 \
 ```
 
 ### Submit with PVC
-#### New PVC bounded to the workspace - will be deleted when the workload is deleted
+#### New PVC bounded to the workspace
+ New PVCs will be deleted when the workload is deleted
```
Comment (SherinDaher-Runai) [inline docs/Researcher/cli-reference/new-cli/cli-examples.md:111]: New PVCs will be deleted when the workload is deleted.

File: docs/Researcher/cli-reference/new-cli/cli-examples.md [hunk 2]
```diff
@@ -107,22 +107,25 @@ runai  mpi submit dist1 --workers=2 -g 1 \
...
 ```shell
 
 runai workspace submit -i ubuntu --new-pvc claimname=yuval-3,size=10M,path=/tmp/test
 ```
-#### New ephemeral PVC - will deleted when the workload is deleted or paused
+#### New ephemeral PVC 
+New ephemeral PVCs will deleted when the workload is deleted or paused
```
Comment (SherinDaher-Runai) [inline docs/Researcher/cli-reference/new-cli/cli-examples.md:117]: New ephemeral PVCs will be deleted when the workload is deleted or paused.

File: docs/Researcher/cli-reference/new-cli/cli-examples.md [hunk 3]
```diff
@@ -107,22 +107,25 @@ runai  mpi submit dist1 --workers=2 -g 1 \
...
 ```shell
 
 runai workspace submit -i ubuntu --new-pvc claimname=yuval2,size=10M,path=/tmp/test,ephemeral
 ```
-#### Existing PVC - will not deleted when the workload is deleted
+#### Existing PVC
+Existing PVCs will not deleted when the workload is deleted
```
Comment (SherinDaher-Runai) [inline docs/Researcher/cli-reference/new-cli/cli-examples.md:123]: Existing PVCs will not be deleted when the workload is deleted.

File: docs/Researcher/cli-reference/new-cli/cli-examples.md (hunk 1 above)
Comment (lavianalon) [inline docs/Researcher/cli-reference/new-cli/cli-examples.md:111]:
```
New PVCs will be deleted when the workload is deleted.
```

File: docs/Researcher/cli-reference/new-cli/cli-examples.md (hunk 2 above)
Comment (lavianalon) [inline docs/Researcher/cli-reference/new-cli/cli-examples.md:117]:
```
New ephemeral PVCs will be deleted when the workload is deleted or paused.
```

File: docs/Researcher/cli-reference/new-cli/cli-examples.md (hunk 3 above)
Comment (lavianalon) [inline docs/Researcher/cli-reference/new-cli/cli-examples.md:123]:
```
Existing PVCs will not be deleted when the workload is deleted.
```

**Review by SherinDaher-Runai — APPROVED** [review]

**Pull request merged**